test/dsl/dsl_basic_model_construction.jl

```diff
@@ -432,11 +491,74 @@ let
     @test length(species(test_network)) == 1
     @test length(parameters(test_network)) == 0
     @test reactions(test_network)[1].rate == ℯ
+end
+
+### Error Test ###
+
+# Erroneous `@reaction` usage.
+let
+    # Bi-directional reaction using the `@reaction` macro.
+    @test_throws Exception @eval @reaction (k1,k2), X1 <--> X2
 
-    @test_throws LoadError @eval @reaction im, 0 --> B
-    @test_throws LoadError @eval @reaction nothing, 0 --> B
-    @test_throws LoadError @eval @reaction k, 0 --> im
-    @test_throws LoadError @eval @reaction k, 0 --> nothing
```

Comment:
forbidden symbol tests now tested more extensively in a test block two blocks down ("# Check that forbidden symbols correctly generates errors"